lib/path.c

```diff
@@ -1020,11 +1020,15 @@ static int ul_path_cpuparse(struct path_cxt *pc, cpu_set_t **set, int maxcpus, i
 {
 	FILE *f;
 	size_t setsize, len = maxcpus * 7;
-	char buf[len];
+	char *buf;
 	int rc;
 
 	*set = NULL;
 
+	buf = malloc(len);
```

Comment:
Just a question. Where is the `buf` released?

Reply:
Nowhere...
Too early in the morning :-|

Reply:
Fixed, thanks for spotting it!